app/assets/javascripts/alchemy/alchemy.sitemap.js.coffee

```diff
@@ -8,7 +8,7 @@ Alchemy.Sitemap =
     @search_field = $("#search_field")
     @filter_field_clear = $('.js_filter_field_clear')
     @display = $('#page_filter_result')
-    @sitemap_wrapper = $('#sitemap-wrapper .sitemap-load-notice')
+    @sitemap_wrapper = $('#sitemap-wrapper')
```

Comment:
That's because the same code also replaces a part of the tree when folding. It's a lot less JS plumbing to just replace the entire unlinked list that's being folded. Now we don't need to disable the spinner, for example. Now folding is basically the same as rendering the whole tree. Just from a different root.